www/src/views/user.ts

```diff
@@ -86,31 +89,30 @@ class UserLink implements ξ.ClassComponent {
 }
 
 export class UserFolloweeList implements ξ.ClassComponent {
-    followees: Array<Followee> = [];
     username: string;
+    followees: Array<Followee> = [];
 
-    async getFolloweeList(username: string) {
+    async getFolloweeList() {
       return Network.request<Array<Followee>>('GET', '/user/' +
-            username + '/followees')
+            this.username + '/followees')
           .then((result: any) => {
             this.followees = result;
-            console.log(this.followees);
-            console.log('getfriendlist');
           });
     }
 
-    capitalize(string) {
-      return string.charAt(0).toUpperCase() + string.slice(1);
-    };
-
     oncreate({attrs}) {
-      console.log('userfolloweelist, on create');
       this.username = attrs.username;
       this.getFolloweeList(this.username);
     }
 
+    onupdate({attrs}) {
+      if (attrs.username != this.username) {
```

Comment:
So this is what will dissappear in Mithril v2, as the onupdate is ran async and also the request, there won't be any infinite request loop.

Reply:
There is support for key:ing so the whole component is reloaded when the `username` changes, but that isn't efficient, right? :smirk: